src/app/messages.ts

```diff
@@ -18,13 +34,19 @@ export const errMessages = {
       "empty": "Whoops... no achievement has been setup ..."
     },
     assessments: {
-      "error": "Whoops... no assessment has been setup ..."
+      "empty": "Whoops... no assessment has been setup ..."
     }
   },
   DirectLink: {
-    
+    "mismatch": "Oops... An error occurred. Please login by typing email and password."
   },
   Events: {
+    bookEvents: {
+      "book": "Whoops, book event process is failed, please try again later."
+    },
+    cancelBooking: {
+      "cancel": "Whoops, cancel booking event process is failed, please try again later."
```

Comment:
Whoops, we could not cancel your booking for this event, please try again later.